tests/k6/src/tests/availability-check.js

```diff
@@ -0,0 +1,33 @@
+/*
+    Test script of platform notifications api with org token
+    Command:
+    docker-compose run k6 run /src/tests/eformidling.js `
```

Comment:
This is a bit outdated now, docker-compose I mean.

Reply:
changed all mentions to podman compose instead